CelesteNet.Client/Components/CelesteNetEmojiComponent.cs

```diff
@@ -46,6 +46,8 @@ public class NetEmojiAsset : ModAsset, IDisposable {
                 Type = typeof(Texture2D);
                 Format = "png";
                 PathVirtual = $"emoji/{ID}";
+                Emoji.Register(ID, GFX.Misc["whiteCube"]);
+                Emoji.Fill(CelesteNetClientFont.Font);
```

Comment:
While this is a good change, this alone might still be too late to fix that one text measuring race condition that you encountered. Is there any other fix to that edge case that I missed? Or do you have any other plans, f.e. emoji packet -> player list regen?

Reply:
Hmmm, well the fix for my bug was only about font measurements of unregistered emoji, i.e. the font saying "`:celestenet_avatar_12:`" is as wide as this literal string is wide instead of 64px. The rendering of the avatars isn't actually an issue, since even without a rebuild, during `Render()` the font will draw the avatar as soon as it's registered.

The only issue I see realistically is "whiteCube" not being exactly as wide/tall as the avatar will be.
I can add a `Handle(CelesteNetConnection con, DataNetEmoji netemoji)` in the player list component, but even then I couldn't ensure this (or the last fragment of it) gets handled here _after_ the emoji has registered itself, can I?

I'll add a `bool ShouldRebuild` to the player list and set that to true if a `DataNetEmoji` gets handled that has `MoreFragments == false` under the assumption that by the time `Update()` gets called a new avatar emoji has been registered and the list should be rebuilt.